main_test.go

```diff
@@ -86,6 +86,7 @@ func TestEndpoints(t *testing.T) {
 		{"/search?prerelease=true", "/search", "search-package-prerelease.json", searchHandler(indexer, testCacheTime)},
 		{"/search?prerelease=foo", "/search", "search-package-prerelease-error.txt", searchHandler(indexer, testCacheTime)},
 		{"/search?category=datastore&prerelease=true", "/search", "search-category-datastore-prerelease.json", searchHandler(indexer, testCacheTime)},
+		{"/search?type=input", "/search", "search-input-packages.json", searchHandler(indexer, testCacheTime)},
```

Comment:
Maybe we should add test cases for:
* An input package with multiple versions.
* A package that has a type on one version, and another type on another version.

Reply:
> An input package with multiple versions.

I added one more revision of `sql_input`.

> A package that has a type on one version, and another type on another version.

`integration_input` package, older revision is integration, newer is input.

I haven't noticed anything unusual, but feel free to review API responses in case I missed something.

Reply:
Good idea, fixed